lib/validators/options_processor/deepValidation.js

```diff
@@ -18,7 +19,24 @@ class DeepValidation {
   static async checkIfIssueHaveProperty (context, issues, property) {
     const output = []
     for (let i = 0; i < issues.length; i++) {
-      const issue = await getIssue(context, issues[i])
+      let issue
+      try {
+        issue = await getIssue(context, issues[i])
+      } catch (err) {
+        // simply skip if the error is 404 (Not Found)
+        if (err.status !== 404) {
+          const errorLog = {
+            log_type: logger.logTypes.ISSUE_GET_FAIL_ERROR,
+            event_id: context.eventId,
+            repo: context.payload.repository.full_name,
+            validator_name: property
+          }
+
+          this.log.info(JSON.stringify(errorLog))
+          throw err
```

Comment:
I don't think we should throw here since you are already logging the error.

Reply:
addressed